### PR TITLE
decorator of wsgi router

### PR DESCRIPTION
**Pull request by RyanKung**
For pull request https://github.com/quantmind/pulsar/pull/277

### CODE REVIEW

File: examples/pulsarapp/manage.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+blueprint = Router('/')
+
+
+@blueprint.router('sync', methods=['get', 'post'])
+def sync_case(request):
+    return request.response('sync')
+
+
+@blueprint.router('async', methods=['delete', 'put'])
+def async_cast(request):
```
Comment (lsbardel) [inline examples/pulsarapp/manage.py]: why is this async? maybe you forgot the `async` keyword?
```python
async def async_cast(request):
     return request.response('async')
```
to test async handlers?

File: pulsar/apps/wsgi/routers.py [hunk 2]
```diff
@@ -276,6 +276,25 @@ def __init__(self, rule, *routes, **parameters):
             self.add_child(self.make_router(rule, method=method,
                                             handler=handler, **rparameters))
 
+    def router(self, rule, methods=['get']):
+        '''Map a function to :class:`Router` and add to the :attr:`routes` list.
+
+        Typical usage:
+
+        app = WsgiApplication('/')
```
Comment (lsbardel) [inline pulsar/apps/wsgi/routers.py]: this does not look correct, should that be ``Router('/')``?

**Comment by RyanKung** [comment]
Fixed, sorry for that mistakes, all OK now. 
Thanks for the reviewing @lsbardel 

```
python setup.py test -a pulsarapp

python setup.py test -a wsgi.router
```

**Comment by lsbardel** [comment]
thanks for this, pulsar 2.0 will be released sometimes in September/October I hope

**Pull request merged**